src/pages/home/report/ReportActionItem.js

```diff
@@ -679,7 +679,7 @@ function ReportActionItem(props) {
             >
                 {(hovered) => (
                     <View style={highlightedBackgroundColorIfNeeded}>
-                        {props.shouldDisplayNewMarker && <UnreadActionIndicator reportActionID={props.action.reportActionID} />}
+                        {props.shouldDisplayNewMarker && ReportActionsUtils.shouldShowNewMarker(props.action) && <UnreadActionIndicator reportActionID={props.action.reportActionID} />}
```

Comment:
Instead of here, can we add this logic in ReportActionsList component instead where we determine if we should display marker or not. Just to keep all logic at same place

Reply:
@MonilBhavsar 

In this case, the new marker would be at the next message(if exists). Is this expected?

Reply:
@MonilBhavsar Can you please check the PR again? I moved the logic to ReportActionsList

Reply:
Here is the best place to add IMO https://github.com/Expensify/App/pull/33162/files#diff-9ac76d645ad53156c86195c1a8f5adb3bb9b71f78351e43c4e1b6d6896921c4eR336

Reply:
@MonilBhavsar 
Can you please check if the link is correct?

Reply:
It was updated after merging main
https://github.com/Expensify/App/pull/33162/files#diff-9ac76d645ad53156c86195c1a8f5adb3bb9b71f78351e43c4e1b6d6896921c4eR351

Reply:
@MonilBhavsar 

In that case, unread marker would move to the next message(if exists) when deleting a message with unread marker. Is this an expected behavior?

Reply:
Yes expected!